Extension/src/common.ts

```diff
@@ -356,15 +387,15 @@ export function resolveVariables(input: string, additionalEnvironment: {[key: st
                 }
                 default: { assert.fail("unknown varType matched"); }
             }
-            return (isString(newValue)) ? newValue : match;
```

Comment:
again, not sure why this one needs to change

Reply:
Same answer as above.  newValue can only be a string, or undefined.  So removed isString() for simplicity.